File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/Model/Item.java
package com.ahf.antwerphasfallen.Model;

/**
 * Created by Jorren on 28/11/2018.
 */

public class Item {
    public static String TYPE_INGREDIENT = "ingredient";
    public static String TYPE_ITEM = "item";

    private int id;
    private String name;
    private String description;
    private String type;

    public Item() {
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getType() {
        return type;
    }
}

File: src/AHF-API/DataLayer/Model/InventoryModel/Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model.InventoryModel
{
    public class Item
    {
        public static string TYPE_INGREDIENT { get; } = "ingredient";
        public static string TYPE_ITEM { get; } = "item";
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Type { get; set; }
    }
}
